Core/MIPS/x86/CompBranch.cpp

```diff
@@ -787,6 +787,10 @@ void Jit::Comp_Syscall(MIPSOpcode op)
 	RestoreRoundingMode();
 	js.downcountAmount = -offset;
 
+	if (!js.inDelaySlot) {
```

Comment:
It's not the break - the issue is this:

```c++
		MIPS_MAKE_LUI(MIPS_REG_A0, 0),
		MIPS_MAKE_SYSCALL("sceGe_user", "sceGeDrawSync"),
```

Since this isn't in a "jr ra" delay slot (like 99.99% of syscalls), nothing actually wrote the PC... and so when the syscall exits, it actually returned to the start of the jit or ir block, instead of the next instruction.

-[Unknown]